c/include/libsbp/integrity_macros.h

```diff
@@ -452,6 +452,38 @@
 
 #define SBP_ACKNOWLEDGE_INTEGRITY_NOT_REQUESTED (0)
 #define SBP_ACKNOWLEDGE_INTEGRITY_REQUESTED (1)
+#define SBP_ACKNOWLEDGE_INTERMEDIATE_CERTIFICATE_MASK (0x1u)
+#define SBP_ACKNOWLEDGE_INTERMEDIATE_CERTIFICATE_SHIFT (9u)
+#define SBP_ACKNOWLEDGE_INTERMEDIATE_CERTIFICATE_GET(flags)                 \
+  ((u16)((u16)((flags) >> SBP_ACKNOWLEDGE_INTERMEDIATE_CERTIFICATE_SHIFT) & \
+         SBP_ACKNOWLEDGE_INTERMEDIATE_CERTIFICATE_MASK))
+#define SBP_ACKNOWLEDGE_INTERMEDIATE_CERTIFICATE_SET(flags, val)           \
+  do {                                                                     \
+    (flags) = (u16)(                                                       \
+        (flags & (~(SBP_ACKNOWLEDGE_INTERMEDIATE_CERTIFICATE_MASK          \
+                    << SBP_ACKNOWLEDGE_INTERMEDIATE_CERTIFICATE_SHIFT))) | \
+        (((val) & (SBP_ACKNOWLEDGE_INTERMEDIATE_CERTIFICATE_MASK))         \
+         << (SBP_ACKNOWLEDGE_INTERMEDIATE_CERTIFICATE_SHIFT)));            \
+  } while (0)
+
+#define SBP_ACKNOWLEDGE_INTERMEDIATE_CERTIFICATE_NOT_REQUESTED (0)
+#define SBP_ACKNOWLEDGE_INTERMEDIATE_CERTIFICATE_REQUESTED (1)
+#define SBP_ACKNOWLEDGE_CERTIFICATE_CHAIN_MASK (0x1u)
+#define SBP_ACKNOWLEDGE_CERTIFICATE_CHAIN_SHIFT (10u)
```

Comment:
Maybe a nitpick, but can you change your PR to use "Certificate Whitelist" instead of "Certificate Chain"? Here and remaining parts of the PR.

Mainly to keep internal nomenclature between this PR and [this one](https://github.com/swift-nav/phoebe/pull/480/files)

Reply:
Hmm the actual message is called `MsgCertificateChain` though, so I kind of feel like we should change other thing, or rename the message

Reply:
I missed that reviewing the other PR. I think you are right, I'll update my PRs and approve this one.